pkg/api/types.go

```diff
@@ -1873,3 +1873,11 @@ type ComponentStatusList struct {
 
 	Items []ComponentStatus `json:"items"`
 }
+
+type ServiceIPAllocation struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
+
+	CIDR string `json:"cidr"`
```

Comment:
I just read descriptions, and I misinterpretted this.  I though you would do one per service, but this is just the state serialized.